README.md

```diff
@@ -63,6 +63,13 @@ and the following CPU types:
 * Storage Admin
 * Vertex AI Administrator
 
+# Prerequisites
```

Comment:
Please add information about the required version of Python and `gcloud`

Reply:
https://github.com/AI-Hypercomputer/xpk/pull/204